tutorials/conf.py

```diff
@@ -70,7 +70,7 @@
 html_sourcelink_suffix = ''
 exclude_patterns = ['_build', '**.ipynb_checkpoints']
 
-cell_timeout = int(os.getenv('QISKIT_CELL_TIMEOUT', 180))
+cell_timeout = int(os.getenv('QISKIT_CELL_TIMEOUT', 360))
```

Comment:
Since I assume these will be built by the main qiskit tutorials process then I guess this change would have to be reconciled  with what is expected there too.